### PR TITLE
Added new solution with Nancy building for net451

### PR DESCRIPTION
**Pull request by khellang**

### CODE REVIEW

**Comment by Yantrio** [comment]
What's the plan with the shared assemblyinfo?
Looks like this could be a solution https://github.com/aspnet/Home/wiki/Project.json-file#shared-files

File: global.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "projects": [ "src", "test", "samples" ],
+    "sdk": {
+        "version": "1.0.0-rc1-final"
```
Comment (Yantrio) [inline global.json]: How come you chose final and not update1?

Comment (khellang) [inline global.json]: Cause it's a long time ago and they're basically the same :smile:

**Comment by khellang** [comment]
> What's the plan with the shared assemblyinfo?

What for? Versioning is handled by the build, so there's no need for patching assembly info. We can just use something like `DNX_BUILD_VERSION` (or the dotnet equivalent)

All the other info is duplicated in project.json for now, because there's no support for pulling up author info etc. in global.json, yet.

The SharedAssemblyInfo.cs is still around because this will just be an addition to the csproj based system, not a replacement yet. There's still loads to do before this is a working alternative.

**Comment by thecodejunkie** [comment]
@Yantrio our `SharedAssemblyInfo.cs` is managed by our build script and really  never manually edited. We issue `rake nancy_build[<version_number_goes_here>]` and it updates it for us and does other magic :D

**Comment by thecodejunkie** [comment]
So this is just to get our current `v2`-target running on both the old and the new project format. It'll be a branching point :D

**Comment by Yantrio** [comment]
Okay, everything else LGTM :+1:

**Comment by khellang** [comment]
Alright. That's it for me :wink: Good look with the CoreCLR porting :grin:

**Comment by thecodejunkie** [comment]
image

**Pull request merged**